pages/login.test.js

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react';
+
+import { loginFormTestIds } from '@/components/forms/Login';
+import Login, { loginTestIds } from './login';
+
+describe(Login, () => {
```

Comment:
Should we also use the string here?

```suggestion
describe('Login', () => {
```

Reply:
done in https://github.com/tyrro/react-survey/pull/50/commits/5d4ec78395abc8fc562ce8a0d93222f71acd4ad9